pkg/kubectl/describe/versioned/describe.go

```diff
@@ -696,7 +696,9 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 		if len(pod.Status.Message) > 0 {
 			w.Write(LEVEL_0, "Message:\t%s\n", pod.Status.Message)
 		}
+		// remove when .IP field is depreciated
 		w.Write(LEVEL_0, "IP:\t%s\n", pod.Status.PodIP)
```

Comment:
Maybe we should hide this inside the descripPodIPs(), so it does something like:

```
describePodIPs() {
  if gate disabled {
    w.Write(LEVEL_0, "IP:\t%s\n", pod.Status.PodIP)  // existing output
    return
  }
  if len(podIPs) == 0 {
    write ("IPs:" ...) using podIP singular field
    return
  }
  // else write plural
```

Reply:
i didn't know that `kubectl` has feature gates. Do you have a pointer to how to configure them?

Reply:
you're right.  My bad